interactions/api/gateway/client.py

```diff
@@ -487,6 +488,8 @@ def __modify_guild_cache():
                     if id:
                         old_obj = _cache.pop(id)
                         self._dispatch.dispatch(f"on_{name}", old_obj)
+                    elif "_delete_bulk" in name:
+                        self._dispatch.dispatch(f"on_{name}", obj)
```

Comment:
This is handled already handled below no?

Reply:
No, since the event **does** contain `_delete` but it does not have a proper `id` attribute.